### PR TITLE
[BUG] - Anchor tags inside opened accordion if used in a flyout are focusable/tabbable though it's hidden (Issue/2799)

### LINKED ISSUE #2799: [BUG] - Anchor tags inside opened accordion if used in a flyout are focusable/tabbable though it's hidden
### Bug description
If the `accordion` component with an anchor link is used as a slot in the `flyout` component, this anchor link is still focusable in the following scenario:

- `flyout`is hidden
- `accordion` has state `open`

This only occurs on links slotted inside an `accordion`. If just an anchor tag is provided as a slot inside the `flyout` it doesn't get focused. Also the `accordion` itself doesn't get focused, though it's a focusable element.

### JS Framework

all

### Porsche Design System version

3.7.0

### What browsers are you seeing the problem on?

all

### Browser version

latest

### Notes
- `inert` attribute (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert) might be a solution to solve the issue 

### Reproduction Example

https://stackblitz.com/edit/twyku1?file=index.html

### Steps to reproduce the behavior

1. Go to Stackblitz example
2. Tab the links and look at the missing focus and the link url preview in the bottom of your browser
3. You will see, that the anchor link inside the flyout > accordion gets focused even when the flyout is hidden

### Expected behavior

All interactive element inside a flyout are not focusable if the flyout is hidden / `<p-flyout open="false" …`.

### PR DESCRIPTION
**Pull request by denyo**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/issue/2799/

#### Scope

Short description of the pull request scope

#### Resolved Issue

Resolves #2799

### CODE REVIEW

File: packages/components/src/components/modal/modal.tsx [hunk 1]
```diff
@@ -158,6 +158,9 @@ export class Modal {
             ...parseAndGetAriaAttributes(this.aria),
           })}
           tabIndex={-1}
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          /* @ts-ignore */
+          inert={this.open ? null : true} // prevents focusable elements within nested open accordion
```
Comment (stephanschroeter) [inline packages/components/src/components/modal/modal.tsx:163]: Wouldn't it be better to set the attribute on the host element to be absolutely sure everything inside the component can't be focused?

File: packages/components-js/tests/e2e/puppeteer/specs/__snapshots__/default-dom.e2e.ts.snap [hunk 2]
```diff
@@ -137,7 +137,7 @@ exports[`should have no basic DOM regression for p-flex-item 1`] = `
 `;
 
 exports[`should have no basic DOM regression for p-flyout 1`] = `
-"<div class="root" role="dialog" aria-modal="true" aria-hidden="true" tabindex="-1">
+"<div class="root" role="dialog" aria-modal="true" aria-hidden="true" tabindex="-1" inert="">
```
Comment (o-hook) [inline packages/components-js/tests/e2e/puppeteer/specs/__snapshots__/default-dom.e2e.ts.snap:140]: Could you get rid of the empty value? `inert` has no value at all and it looks a bit weird with empty "".

Comment (denyo) [reply packages/components-js/tests/e2e/puppeteer/specs/__snapshots__/default-dom.e2e.ts.snap:140]: I don't think so. That's in the nature of jsx.

File: packages/components/src/components/modal/modal.tsx (hunk 1 above)
Comment (denyo) [reply packages/components/src/components/modal/modal.tsx:163]: Probably makes no difference but adds the chance that the attribute can be removed.

**Review by stephanschroeter — APPROVED** [review]

**Pull request merged**